# Agent ID
int32 id

# Human-readable name
string name

# ID of object representing the agent
int32 avatar_id
